react/features/rtcstats/middleware.ts

```diff
@@ -118,8 +118,25 @@ MiddlewareRegistry.register((store: IStore) => (next: Function) => (action: AnyA
     }
     case TRACK_UPDATED: {
         if (canSendRtcstatsData(state)) {
-            const { videoType, jitsiTrack } = action?.track || { };
-            const { ssrc } = jitsiTrack || { };
+            const { videoType, jitsiTrack, muted } = action?.track || { };
+            const { ssrc, isLocal, videoType: trackVideoType, conference } = jitsiTrack || { };
+
+            if (trackVideoType === 'camera' && conference) {
+                if (muted && isLocal()) {
+                    RTCStats.sendFaceLandmarksData({
+                        duration: 0,
+                        faceLandmarks: 'camera-off',
```

Comment:
Can't you simplify this by having a isLocal check, then send one event and compute the name with a ternary operator?

Reply:
changed it.